saleor/plugins/webhook/tasks.py

```diff
@@ -159,21 +159,30 @@ def send_webhook_request(webhook_id, target_url, secret, event_type, data):
     message = data.encode("utf-8")
     signature = signature_for_payload(message, secret)
     if parts.scheme.lower() in [WebhookSchemes.HTTP, WebhookSchemes.HTTPS]:
-        send_webhook_using_http(target_url, message, domain, signature, event_type)
+        try:
+            send_webhook_using_http(target_url, message, domain, signature, event_type)
+        except RequestException as e:
+            logger.debug("[Webhook] Failed request to %r: %r.", target_url, e)
```

Comment:
I don't think that this should be a debug. We will lose information that someone has failed webhook integration.  Currently it is the only way to check this. With debug, we will lose this possibility. 
Also, the celery task uses autoretry_for `RequestException`, which you're handling here, I am not sure if such task will be retried in that case.